apps/documentation/docs/getting-started/serverless.md

```diff
@@ -2,24 +2,286 @@
 
-To get started with running a version of the leadership site locally follow these steps:
+To setup a new serverless service in this repo, follow these steps:
 
-### Setup
+### Install Serverless Framework CLI
 
-- Clone the repo `git clone https://github.com/acmutd/leadership.git`
-- Fill in the values in `.env.local.example` and rename it to `.env.local`. Reach out to an existing contributor to get these values.
+- To install Serverless Framework CLI, run the following:
 
-Note: You can also find the environment variables via [Doppler](https://doppler.com). Open the Doppler website, sign in with your ACM account and open the leadership project. Then copy the variables in the `dev` config to your local `.env.local` file. Currently, the leadership site uses Doppler to fetch environment variables for its production vercel deployment. It is not configured to pull the environment variables from Doppler for local workloads yet.
+```
+npm i -g serverless
+```
 
-### Launch
+### Directory Setup
 
-- Run `npm install`
-- Run `npm run dev`
+- Create a new folder with the name of the service inside the `services` directory
+  - e.g: If your service name is `email`, then create the new directory `services/email`.
+- Run the following command and follow the prompt:
+
+```
+npm init -w ./services/email
+```
+
+:::important
+
+- When run the above command, or any command afterward, make sure to replace `email` with the name of your service.
+
+- When being prompted package name, make sure that it should be in the form of `@acmutd/<service_name>`
 
-:::caution
-If you are using Windows, running `npm run dev` may cause some errors. By default this project is setup to inspect server side logs and print them to the terminal window running the application. If there are any issues remove the `NODE_OPTIONS='--inspect'` from the scripts section of the `package.json` file for `npm run dev`.
 :::
 
-After running these steps you should be able to open up `http://localhost:3000` and view the leadership site!
+### Install Serverless Function Dependencies
+
+- To install the dependencies that will be used by Serverless Framework, run the following command:
+
+```
+npm i -w ./services/email -D serverless-iam-roles-per-function serverless-create-global-dynamodb-table serverless-offline serverless-prune-plugin
+```
+
+- To install AWS Lambda related dependencies, run the following command:
+
+```
+npm i -w ./services/email aws-sdk aws-lambda
+```
+
+- To install Lambda Powertools, run the following command:
+
+```
+npm i -w ./services/email @dazn/lambda-powertools-cloudwatchevents-client @dazn/lambda-powertools-correlation-ids @dazn/lambda-powertools-logger @dazn/lambda-powertools-pattern-basic @dazn/lambda-powertools-lambda-client @dazn/lambda-powertools-sns-client @dazn/lambda-powertools-sqs-client @dazn/lambda-powertools-dynamodb-client @dazn/lambda-powertools-kinesis-client
+```
+
+### Setup Linting
+
+- To install linting dependencies, run the following command:
```

Comment:
Only need to install in root package.json I believe. Might need to test with husky & vscode extension